Lib/lang.php

```diff
@@ -38,33 +38,34 @@
   
   // Error messages
   'er.xnatprovisioner.id.none'        => 'No identifier of type %1$s found for CO Person',
-  'er.coservier.id.none'              => 'No id found for this service',
+  'er.coserver.id.none'              => 'No id found for this service',
```

Comment:
nit: preserve alignment

```suggestion
  'er.coserver.id.none'               => 'No id found for this service',
```